### PR TITLE
Cql4bonnie h1h2

### PR DESCRIPTION
**Pull request by lukoktonos**
Previously, Bonnie used heading tags like `<h1>, <h2>`, etc. to add styling to a
page rather than just denote page division.  In order to be comlpiant with 508
specs, the styling was moved into CSS classes "h1-style", "h2-style", etc. and
the headings were removed except in cases where they actually denote a proper
page division.

Most changes should result in no UI changes, but there was one very minor space change that we couldn't get to be exactly the same on the patient builder page, but should be acceptable:
Before
image
After
image

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1012
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed
- [X] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [X] Tests are included and test edge cases (N/A)
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: https://jira.mitre.org/browse/BONNIE-1118
- [X] Justification for using JIRA tests: WAVE tool cannot be run in a unit test.
- [X] JIRA tests have been added to sprint

**Reviewer 1:** (@hossenlopp)

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:** (@holmesie)

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: app/assets/javascripts/templates/patient_builder/edit_criteria.hbs [hunk 1]
```diff
@@ -32,7 +32,8 @@
         <div class="pull-left">
           <span class="highlight-indicator sr-only"></span>
 
-          <p><strong>{{definition_title}}:</strong> {{description}}</p>
+          {{!-- <p><strong>{{definition_title}}:</strong> {{description}}</p> --}}
```
Comment (holmesie) [inline app/assets/javascripts/templates/patient_builder/edit_criteria.hbs]: Should this comment be removed?

File: app/assets/stylesheets/builder.less [hunk 2]
```diff
@@ -294,7 +321,7 @@
             left: -(@grid-gutter-width / 4) - 1; // width/2 (-1 to account for border)
             .rotate(45deg);
           }
-
+
```
Comment (holmesie) [inline app/assets/stylesheets/builder.less]: Remove this whitespace

File: app/assets/stylesheets/measures.less [hunk 3]
```diff
@@ -1,5 +1,21 @@
 // Dashboard - Measures View
 
+//Mixins
+.h2-style {
```
Comment (holmesie) [inline app/assets/stylesheets/measures.less]: I'm not that well versed in how less works, but I'm pretty sure CSS classes are global. Wouldn't this interfere with the `h2-style` that you defined in `app/assets/stylesheets/builder.less`?

**Review by holmesie — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/measures.less (hunk 3 above)
Comment (lukoktonos) [reply app/assets/stylesheets/measures.less]: You are right--it is a redundant class, and the extra elements from `builder.less` were making their way through to the measures page as well. I removed it and checked that the measure page looked the same.

**Review by holmesie — DISMISSED** [review]

**Review by hossenlopp — CHANGES_REQUESTED** [review]
I found a few visual discrepancies in the dashboard and the patient builder. Will comment with comparison screenshots.

**Comment by hossenlopp** [comment]
Patient builder element headers went pretty wonky. Latest from alpha on the left, this branch on the right. 
screen shot 2017-10-05 at 9 01 24 am

**Comment by hossenlopp** [comment]
Column headers for the measure dashboard have something odd with their vertical alignment. Latests from alpha is on the left, this branch on the right.
screen shot 2017-10-05 at 8 53 12 am

**Comment by lukoktonos** [comment]
These styles should match now.  Oddly on my chrome, I don't see the light gray box behind the column headers on either this branch or bonnie alpha.

**Review by hossenlopp — APPROVED** [review]

**Review by holmesie — APPROVED** [review]

**Pull request merged**